docs/howtouse.md

```diff
@@ -1,9 +1,154 @@
 ## How to use
-Inside your project, there are a few basic prerequisites for vdist to work.
+There are two ways to use vdist:
+
+1. Using its [console launcher](#console-launcher)
+2. [Create a small Python file](#integrating-vdist-in-a-python-script) that actually uses the vdist module
+
+On both cases you are strongly advised to create a requirements.txt
+('pip freeze > requirements.txt' inside a virtualenv should give you a good
+start); nevertheless you are likely to already have one.
+
+### Console launcher
+Probably the simplest way to use vdist. When you install vdist Pypi package a
+vdist executable is installed in your python distribution bin folder from which
+you installed the package. If you have that folder in your PATH then you can
+use vdist command directly, if not add that folder to path or call launcher
+using its absolute path.
+
+If you are going to use console launcher you only need to create a **configuration
+file** to put all parameter for all packages you want to generate. An example of a
+configuration file like that could be:
+
+```ini
+[DEFAULT]
+app = geolocate
+version = 1.3.0
+source_git = https://github.com/dante-signal31/${app}, master
+fpm_args = --maintainer dante.signal31@gmail.com -a native --url
+    https://github.com/dante-signal31/${app} --description
+    "This program accepts any text and searchs inside every IP
+    address. With each of those IP addresses,
+    ${app} queries
+    Maxmind GeoIP database to look for the city and
+    country where
+    IP address or URL is located. Geolocate is designed to be
+    used in console with pipes and redirections along with
+    applications like traceroute, nslookup, etc."
+    --license BSD-3 --category net
+requirements_path = /REQUIREMENTS.txt
+runtime_deps = libssl1.0.0, dummy1.0.0
+compile_python = True
+python_version = 3.4.4
+output_folder = ./vdist
+
+[Ubuntu-package]
+profile = ubuntu-trusty
+
+[Centos7-package]
+profile = centos7
+```
+
+Running vdist with last configuration file will place generated packages at path
+you set in *output_folder* variable. If you don't use absolute path but a
+relative one, then reference folder is the one where you are when vdist command
+is called. **Always set *output_folder* variable**.
+
+As you can see, there are three main **sections** in previous configuration: DEFAULT,
+Ubuntu-package, Centos7-package. You can name each section as you want but
```

Comment:
No, it is not *absolutely* necessary. Actually you can name them just `[ubuntu-trusty]` or whatever you want, but I appended "-package" to make that parameter different than those used in `profile` parameter in order to avoid confusions.